### DISCUSSION TITLE
How to integrate snapshot with EarlGrey2?

### DISCUSSION THREAD

**Discussion by liweijian** [pointfreeco/swift-snapshot-testing#494] [Q&A]
Hi folks, I'm new here and feel this project awesome!

I was trying to integrate snapshot testing with [EarlGrey2](https://github.com/google/EarlGrey/tree/earlgrey2), and couldn't find any docs on it.

For example,

<img src="https://user-images.githubusercontent.com/19320783/127165944-553b039e-556f-4be6-9e4a-cae3608fcc88.png" width=500>

What a want to do is just to capture the screen shot and save it, analysis it later using [screenshotbot](https://github.com/screenshotbot/screenshotbot-oss/).

Let me know if anything that I was wrong or had missed anything :)

**Comment by mbrandonw** [comment]
Hi @liweijian, I don't have an answer for you unfortunately, but I'm going to move this to a discussion since it's not an issue with the library.
